ignek_intranet/modules/employee/employee-web/src/main/java/com/ignek/portal/employee/web/constants/EmployeeWebConstants.java

```diff
@@ -0,0 +1,29 @@
+package com.ignek.portal.employee.web.constants;
+
+/**
+ * @author ignek
+ */
+public class EmployeeWebConstants {
+
+	public static final String EMPLOYEE_WEB_PORTLET_KEY = "com_ignek_portal_employee_web_EmployeeWebPortlet";
+
+	public static final String EMP_ID = "empId";
+
+	public static final String FIRST_NAME = "firstName";
+
+	public static final String LAST_NAME = "lastName";
+
+	public static final String EMAIL_ADDRESS = "emailAddress";
+
+	public static final String PHONE_NUMBER = "phoneNumber";
+
+	public static final String ADDRESS_LINE_1 = "addressLine1";
+
+	public static final String ADDRESS_LINE_2 = "addressLine2";
+
+	public static final String CITY = "city";
+
+	public static final String ZIPCODE = "zipCode";
+
+	public static final String DESIGNATION = "designation";
```

Comment:
These constants are there in ignek_intranet/modules/common/src/main/java/com/ignek/intranet/common/constants/IntranetConstants.java

Please remove these duplicate constants